src/test/java/guitests/ContextMenuTests.java

```diff
@@ -97,4 +106,46 @@ public void test3() {
         sleep(EVENT_DELAY);
     }
 
+    /**
+     * Marks and tests issues in a panel of 10 issues
+     * from Issue #{index} in the UI right to the end of the list as read/unread
+     * @param isMarkAsRead If true, tests whether issues on/below the selected issue are being correctly marked read
+     *                      If false, tests whether issues on/below the selected are being correctly marked unread
+     * @param index The issue number in the panel
+     */
+    private void markIssuesBelow(int index, boolean isMarkAsRead){
+        click("#dummy/dummy_col0_" + index);
+        rightClick("#dummy/dummy_col0_" + index);
+        if (isMarkAsRead){
+            click("Mark all below as read");
+        } else {
+            click("Mark all below as unread");
+        }
+        for (int i = index; i >= 1; i--){
+            verifyReadStatusOfIssue(i, isMarkAsRead);
+        }
+    }
+
+    /**
+     * Tests whether a list panel cell corresponding to a particular index is marked as read/unread
+     */
+    private void verifyReadStatusOfIssue(int index, boolean isExpectedStatusRead){
+        ListPanelCell listPanelCell = find("#dummy/dummy_col0_" + index);
+        if (isExpectedStatusRead){
+            assertTrue(listPanelCell.getIssue().isCurrentlyRead());
+        } else {
+            assertFalse(listPanelCell.getIssue().isCurrentlyRead());
+        }
+    }
+
+    /**
+     * Brings the {index} issue and the issues below that in view for a list panel.
+     */
+    private void bringSelectedAndIssuesBelowInFocus(int index) {
+        click("#dummy/dummy_col0_" + index);
+        for (int i = 0; i < 15; i++) {
+            push(KeyCode.DOWN);
+        }
+    }
```

Comment:
By focus I mean in the view of the panel. I want to scroll down so that issue `index` and all issues below can be seen on the UI of the panel. Basically, move towards the end of the panel.